build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        KOTLIN_VERSION = "1.9.0"
```

Comment:
@sennhann29 greetings to you all I apologize if I have delayed anybody's project 🙏🤴